### DISCUSSION TITLE
Teacher application

### DISCUSSION THREAD

**Discussion by jbergant** [community/community#53590] [GitHub Education]
I'm trying to apply as a teacher.
My school does not issue any teacher id. No matter what kind of document I upload: I get suggestions to scan the teacher ID. Is there any other way to be validated as a teacher?
Kind regards,
Jana

**Comment by samus-aran** [comment]
Hi @jbergant 👋 

We're happy you're here!

You should be able to submit alternative pieces of evidence but sometimes we request students/teachers submit a live photo of their evidence rather than an upload. We understand that students/teachers may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
